restservice/src/main/java/org/itevents/util/mail/MailBuilderUtil.java

```diff
@@ -33,69 +32,78 @@
     @Value("classpath:utils/mailBuilder/UserOtpMail.xsl")
     private Resource emailUserOtpTemplateXslResource;
 
-    public String buildHtmlFromEventsList(List<Event> events) throws ParseException, JAXBException, IOException,
-            TransformerException {
-        return buildMailFromXmlEvents(buildXmlFromEventList(events));
+    public String buildHtmlFromEventsList(List<Event> events) {
+        try {
+            return buildMailFromXmlEvents(buildXmlFromEventList(events));
+        } catch (JAXBException | TransformerException | IOException e) {
+            throw new MailBuilderUtilException(e.getMessage(), e);
+        }
     }
 
-    public String buildHtmlFromUserOtp(User user, OneTimePassword oneTimePassword)  throws Exception {
-        return buildMailFromXmlUserOtp(BuildXmlFromUserOtp(user, oneTimePassword));
+    public String buildHtmlFromUserOtp(User user, OneTimePassword oneTimePassword) {
+        try{
+            return buildMailFromXmlUserOtp(BuildXmlFromUserOtp(user, oneTimePassword));
+        } catch (JAXBException | TransformerException | IOException e) {
+        throw new MailBuilderUtilException(e.getMessage(), e);
```

Comment:
@IgorMaksymov 
1) fix indentation
2) you have to handle `MailBuilderUtilException`